Symfony/CS/Tokenizer/Transformers.php

```diff
@@ -22,9 +22,9 @@
 class Transformers
 {
     /**
-     * Array of registered Transformer classes.
+     * Array of registered classes implementing TransformerInterface.
      *
-     * @var Transformer[]
+     * @var array
```

Comment:
Good point, wasn't sure if that was valid. Updated (will squash the commits at the end)